src/pages/lib/auth/social/q/platform/[platform].mdx

```diff
@@ -1,15 +1,1434 @@
+<InlineFilter filters={["js", "react-native"]}>
 
-<Fragments fragments={{js: js0}} />
+You can enable your users to sign-in and authenticate with your app using their existing accounts with social providers like Apple, Amazon, Facebook, and Google. Once you configure your backend resources, you can then configure your app to sign in users for that provider. This process will securely exchange credentials and gather user information. This information can then be used to create a new user in your app or link the social account to an existing user. Incorporating social provider sign-in provides your users with a streamlined registration, more sign-in options, and an integrated experience across their existing accounts and your app.
 
-import reactnative0 from "/src/fragments/lib/auth/js/react-native-social.mdx";
+In this guide we will review how you can add sign-in with social providers by first setting up your developer account with the provider. After this step, you can then configure your Auth category and integrate social sign-in with your app.
 
-<Fragments fragments={{'react-native': reactnative0}} />
+Before you begin you will need:
+- An Amplify project
+- The Amplify libraries installed and configured
+
+<Accordion title='Review how OAuth authentication works' headingLevel='3' eyebrow='Learn more'>
+
+The preferred way to incorporate social provider sign-in is via an OAuth redirect which lets users sign in using their social media account and creates a corresponding user in the Cognito User Pool. 
+
+[OAuth 2.0](https://en.wikipedia.org/wiki/OAuth) is the common Authorization framework used by web and mobile applications for accessing user information ("scopes") in a limited manner. Common analogies you will hear in OAuth is that of boarding a plane or staying in a hotel - showing your identification is the authentication piece (signing into an app) and using the boarding pass/hotel key is what you are authorized to access.
+
+OAuth support in Amplify uses Cognito User Pools and supports federation with social providers, which will automatically create a corresponding user in the User Pool after they sign in. With this design you do not need to include an SDK for the social provider in your app. After you set up your developer account with these social providers you can configure the `Auth` category by running `amplify add auth` and selecting the social provider option. You can then use `Auth.federatedSignIn()` in your app to either show a pre-built "Hosted UI" or pass in a provider name (e.g. `Auth.federatedSignIn({provider: 'Facebook'})`) to interface directly and build your own UI.
+	
+![Image](/images/SocialAuthZ.png)
+	
+Alternatively, you can also get credentials directly from Identity Pools by passing tokens from a social provider directly with `Auth.federatedSignIn()`. However you will have to use that provider's SDK in your app as well as manage the tokens from the provider and their refresh cycles manually.
```

Comment:
can we remove mentions of this API. It has been an extremely confusing piece for our customers.